prow/Makefile.gcloud.mk

```diff
@@ -41,11 +41,8 @@ configure-docker: activate-serviceaccount
 
 get%api-resources:
 	$(eval PRUNE_WL=$(shell KUBECONFIG=$(KUBECONFIG) \
```

Comment:
There is WAY too much eval everywhere, which violates our style guidelines for bash.

http://google.github.io/styleguide/shellguide.html#s6.6-eval

We should also try to putting logic in makefiles as these are not portable across repos

Why is this necessary here?

Reply:
This is to _set_ to global `PRUNE_WL` `make` variable used in the other `make` deploy rules in `prow/Makefile`, so a single target that can handle all of these. I understand this is difficult to follow. I can work on a revision but I would like to get this specific change to `api-resources.py` in to fix the deploy job.

Reply:
I agree. I think a better approach is to pull this out of the makefilem like you suggest, into a script or program.

Reply:
I added this: https://github.com/istio/test-infra/issues/2552. I don't have time atm to refactor and debug this but will soon.